plugins/modules/proxysql_query_rules.py

```diff
@@ -187,6 +204,25 @@
     state: present
     load_to_runtime: False
 
+# This example adds a caching rule for a query that matches the digest.
+# The query digest can be obtained from the `stats_mysql_query_digest`
+# table. `cache_ttl` is specified in milliseconds. Empty responses are
+# not cached.
+
+- name: Add a cache rule
```

Comment:
This message can also be used without multiplexing. I think that it is not necessary to specify it in the example, since its use is obvious. As far as I can see, the old error_msg parameter was also not specified in the examples. But if you want to add more examples, I can add a few so that they cover more parameters :)

Reply:
I'm not a user of this, so you know better. Generally more examples is a usually good approach, though not tooo many:)

Reply:
No problem, I'll add some more examples covering the new parameters :)